os/wqueue/work_process.c

```diff
@@ -274,7 +274,9 @@ void work_process(FAR struct wqueue_s *wqueue, int wndx)
 #endif
 		/* Wait indefinitely until signalled with SIGWORK */
 		wqueue->worker[wndx].busy = false;
+#ifdef CONFIG_SCHED_WORKQUEUE_SORTING
```

Comment:
This looks strange. This functionality is not related with sorting. This is for waiting new work. Could you check this conditional?

Reply:
I re-check the commit which merged kernel and user workqueue. This is wrong. Could you make a change?

Reply:
But the declaration of this variable(set) was under that specific flag.
Thats the reason, we made the changes accordingly. Pls refer below code

void work_process(FAR struct wqueue_s *wqueue, int wndx)
{
	volatile FAR struct work_s *work;
	worker_t worker;
	FAR void *arg;
	clock_t elapsed;
	clock_t stick;
	clock_t ctick;
	clock_t next;

**#ifdef CONFIG_SCHED_WORKQUEUE_SORTING**
	sigset_t set;
	**sigemptyset(&set);**
	sigaddset(&set, SIGWORK);
#else 
	clock_t remaining;
#endif

Reply:
@satheesh-ks I mean that code is wrong. I know why you add that conditional. But because original is wrong, yours is also wrong. That's why I suggest modify together.

Reply:
@satheesh-ks I sent you messages for that through the chat. Could you check that?